DuckDuckGo/Subscription/Views/SubscriptionSettingsView.swift

```diff
@@ -266,18 +285,6 @@ struct SubscriptionSettingsView: View {
                 .padding(.vertical, -10)
             if configuration == .subscribed || configuration == .expired {
                 devicesSection
-                    .alert(isPresented: $isShowingRemovalNotice) {
```

Comment:
This alert was moved into `removeFromDeviceView` var alongside its `SettingsCustomCell` definition so we have it extracted in the same way as `restorePurchaseView` for easier and cleaner reuse.